test/refactoring/RefactoringKind/basic.swift

```diff
@@ -422,4 +425,6 @@ func testConvertToIfLetExpr(idxOpt: Int?) {
 
 // CHECK-CONVERT-TO-GUARD-EXPRESSION: Convert To Guard Expression
 
-// CHECK-CONVERT-TO-IFLET-EXPRESSION: Convert To IfLet Expression
\ No newline at end of file
+// CHECK-CONVERT-TO-IFLET-EXPRESSION: Convert To IfLet Expression
+
+// CHECK-CONVERT-TO-COMPUTED-PROPERTY: Convert To Computed Property
```

Comment:
It'd be good to test the cases where you're returning `false` in the isApplicable method too with a `MYCHECKPREFIX-NOT:` line making sure "Convert To Computed Property" isn't suggested.

Reply:
I'm not sure how to use -NOT postfix, so I just made explicit list of expecting menu items: CHECK-NONE and CHECK-CONVERT-TO-COMPUTED-PROPERTY2.

Reply:
In case you're interested you can check out the FileCheck documentation [here](https://llvm.org/docs/CommandGuide/FileCheck.html#the-check-not-directive). In this case you'd write:
```
// CHECK-CONVERT-TO-COMPUTED-PROPERTY2: Action begins
// CHECK-CONVERT-TO-COMPUTED-PROPERTY2-NOT: Convert To Computed Property
// CHECK-CONVERT-TO-COMPUTED-PROPERTY2: Action ends 
```
But what you've got works too!